dockerfiles/cli/version/latest.ver

```diff
@@ -1 +1 @@
-5.1.0
\ No newline at end of file
+5.1.1
```

Comment:
I don;t want break things now, I will test this later if it works I will add new line